slakh96/no-mans-land
# Visual Feedback - Spaceship health progress bar as Oxygen (percentage from x deposited/total)

Testing
- Delete previous OxygenMeterCanvas if it exists in your scene
- Drag and drop ONLY HealthBarCanvas from Assets/Prefabs/Healthbars to scene.
- Should work as normal. Test as you like.
